src/main/java/task/NewOpportunityPortal/cryp/EncryptDecrypt.java

```diff
@@ -15,27 +14,27 @@
 import java.security.InvalidKeyException;
 import java.security.NoSuchAlgorithmException;
 
-@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@RequiredArgsConstructor
 @Component
 public class EncryptDecrypt {
 
-    private static final String SECRET_KEY="${secret.key}";
+    private String secretKey=System.getenv("SECRET_KEY");
```

Comment:
It will be better to do like this:
`    @Value("#{environment.SECRET_KEY}")
    private String secretKey;`
And don't forget about indention between characters